install/istio-system/base/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Do we need istio? How/Why is it used? (We do not use istio in ODH at the moment)

Reply:
Currently, there's a link to a kubeflow installation in the UI.  In the ODH and Kubeflow on OpenShift docs it recommends finding the route to the Kubeflow UI using:
```shell script
oc get routes -n istio-system istio-ingressgateway -o jsonpath='http://{.spec.host}/'
```
In order to reproduce this I needed a role with access to that route and I was trying to be as tight as possible.  I'm open to alternatives or maybe we can be loose for now and use a `ClusterRole`.

Reply:
I don't think this is likely to work for our ODH installs.  As noted, the ODH install doesn't include istio.  The route being fetched here would be to the Kubeflow centraldashboard, I think, which can be installed if folks really wanted it, but is not part of what we would consider a typical ODH install.

Reply:
@vpavlin @crobby shall I just remove all the Kubeflow stuff since it doesn't get installed via the ODH operator anyway?

Reply:
Alright, I'm just going to delete Kubeflow for now and revisit when we want to show it.